VAMobile/src/api/claimsAndAppeals/uploadFileToClaim.tsx

```diff
@@ -82,22 +86,34 @@ const uploadFileToClaim = ({ claimID, request, files }: UploadFileToClaimParamat
  */
 export const useUploadFileToClaim = (
   claimID: string,
-  request: ClaimEventData,
+  request: ClaimEventData | undefined,
   files: Array<Asset> | Array<DocumentPickerResponse> | undefined,
 ) => {
   const queryClient = useQueryClient()
   const dateUploadedString = DateTime.local().toISO()
   const claimData = queryClient.getQueryData([claimsAndAppealsKeys.claim, claimID]) as ClaimData
-  const indexOfRequest = claimData.attributes.eventsTimeline.findIndex((el) => el.description === request.description)
-  claimData.attributes.eventsTimeline[indexOfRequest].uploaded = true
-  claimData.attributes.eventsTimeline[indexOfRequest].status = FILE_REQUEST_STATUS.SUBMITTED_AWAITING_REVIEW
-  claimData.attributes.eventsTimeline[indexOfRequest].documents = createFileRequestDocumentsArray(
-    files || [],
-    request?.trackedItemId || undefined,
-    request?.documentType || '',
-    dateUploadedString,
-  )
-  claimData.attributes.eventsTimeline[indexOfRequest].uploadDate = dateUploadedString
+  if (request) {
+    const indexOfRequest = claimData.attributes.eventsTimeline.findIndex((el) => el.description === request.description)
+    claimData.attributes.eventsTimeline[indexOfRequest].uploaded = true
+    claimData.attributes.eventsTimeline[indexOfRequest].status = FILE_REQUEST_STATUS.SUBMITTED_AWAITING_REVIEW
+    claimData.attributes.eventsTimeline[indexOfRequest].documents = createFileRequestDocumentsArray(
+      files || [],
+      request?.trackedItemId || undefined,
+      request?.documentType || '',
+      dateUploadedString,
+    )
+    claimData.attributes.eventsTimeline[indexOfRequest].uploadDate = dateUploadedString
+  } else {
+    const claimEvent: ClaimEventData = {
+      type: '',
+      date: dateUploadedString,
+      status: FILE_REQUEST_STATUS.SUBMITTED_AWAITING_REVIEW,
+      uploaded: true,
+      uploadDate: dateUploadedString,
+      documents: createFileRequestDocumentsArray(files || [], undefined, '', dateUploadedString),
```

Comment:
blank type again.

Reply:
same as above this is for us locally so we can display the file in the list so it can be nil